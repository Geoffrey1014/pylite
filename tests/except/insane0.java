class Return1 {
    public static void main(String[] args) throws Exception
    {
        try {
            throw new Exception("Exception");
        } 
        finally {
            System.out.println("finally");
        }
    }
}
