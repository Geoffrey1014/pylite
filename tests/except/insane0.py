def main():
    try:
        print 'try'
        0/0
    finally:
        print 'finally' 

print 'global start'
main()
print 'global end'
