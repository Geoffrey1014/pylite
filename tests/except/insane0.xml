lambda
  name __init__
  param_count 0
  local_count 0
  label_count 0
  insn_vector
    load_const c0
    store_global c1
    load_const c2
    print
    load_global c1
    call 0
    pop
    load_const c3
    print
    load_const c4
    return
  const_vector
    lambda
      name main
      param_count 0
      local_count 1
      label_count 4
      insn_vector
        label L0
        load_const c0
        print
        load_const c1
        load_const c1
        div
        pop
        label L1
        jmp L3
        label L2
        store_local -1
        load_const c2
        print
        finally_exit -1
        label L3
        finally_call L2
        load_const c3
        return
      const_vector
        "try"
        0.000000
        "finally"
        None
      guard_vector
        guard
          start L0
          end L1
          handler L2
      line_number_map 3 3 3 4 4 4 4 3 3 3 3 6 6 3 3 3 1 1
    main
    "global start"
    "global end"
    None
  guard_vector
  line_number_map 1 1 8 8 9 9 9 10 10 1 1
