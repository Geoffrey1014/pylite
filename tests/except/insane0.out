global start
try
finally
ZeroDivisionError:
  At insane0.py, line 4, in main()
  At insane0.py, line 9, in __init__()
